FHEM/lib/signalduino_protocols.hash

```diff
@@ -2078,11 +2078,11 @@
 			sync         => [1,-36],
 			clockabs     => 280,						# -1=auto	
 			format       => 'twostate',
-			preamble     => 'u90#',			
+			preamble     => 'P90#',
 			length_min   => '33',
 			length_max   => '36',
-			#clientmodule	=> 'IT',
-			#modulematch	=> '^P90#.*',
+			clientmodule => 'SD_UT',
+			modulematch	=> '^P90#.*',
 			developId    => 'y',
```

Comment:
Das könnte und sollte entfallen oder?

Reply:
Was meinst du genau mit entfallen?

Reply:
Ich meine, wenn es ein logisches Modul gibt. Braucht es dann noch developid=

Reply:
Sollte das developId => 'y' stören, so nehme ich es auch heraus und der User wurde ja hingewiesen in der Commandref, das nur Empfang möglich ist ;-)

Reply:
Wenn die Protokoll Definition selbst nicht mehr in Entwicklung ist, brauchen wir das Flag nicht und sollten es entfernen.

Reply:
@sidey79 
Bitte nich verwechseln :-)

Wir haben
`# developId   => 'm'       # logical module is under development`
und weil das Protokoll nicht vollständig decodierbar ist derzeit aufgrund von fehlenden Ideen oder Erkenntnis was freie veränderbare bits sind (welche man zum senden aber benötigt), so ist deswegen
`# developId   => 'y'       # protocol is under development, all IDs in the development attribute with developId => 'y' are used
`
gesetzt wurden von mir.

Reply:
@HomeAutoUser 

Irgendwas scheint unklar. Ich versuche mal meine Sicht zu beschreiben

Der Eintrag "90" im Protocol_Hash ist für die Demodulation / Modulation verantwortlich.
Die demodulierten Daten werden an das SD_UT Modul übergeben.

Aus dieser Sicht heraus, denke ich, dass das "Protokoll" mit der ID "90" nicht mehr in Entwicklung ist, was nicht gleichbedeutend damit ist, dass im SD_UT Modul noch beim Interpretieren der Bits offen ist.

Reply:
> Wenn die Protokoll Definition selbst nicht mehr in Entwicklung ist, brauchen wir das Flag nicht und sollten es entfernen.

Sehe ich genauso, hier ist das developId => 'y' nicht mehr notwendig.
Du kannst ja, im Comment der Protokolldefinition reinschreiben, daß das Senden noch nicht funktioniert.

Reply:
Nachbesserung wurde durchgeführt.